docs/source/jit_language_reference.rst

```diff
@@ -0,0 +1,438 @@
+     - Not Supported
+     -
+   * - `6.2.9. Yield expressions <https://docs.python.org/3/reference/expressions.html#yield-expressions>`_
+     - Not Supported
+     -
+   * - `6.2.9.1. Generator-iterator methods <https://docs.python.org/3/reference/expressions.html#generator-iterator-methods>`_
+     - Not Supported
+     -
+   * - `6.2.9.2. Examples <https://docs.python.org/3/reference/expressions.html#examples>`_
+     - Not Supported
+     -
+   * - `6.2.9.3. Asynchronous generator functions <https://docs.python.org/3/reference/expressions.html#asynchronous-generator-functions>`_
+     - Not Supported
+     -
+   * - `6.2.9.4. Asynchronous generator-iterator methods <https://docs.python.org/3/reference/expressions.html#asynchronous-generator-iterator-methods>`_
+     - Not Supported
+     -
+   * - `6.3. Primaries <https://docs.python.org/3/reference/expressions.html#primaries>`_
+     - Supported
+     -
+   * - `6.3.1. Attribute references <https://docs.python.org/3/reference/expressions.html#attribute-references>`_
+     - Partially Supported
+     - Attribute references on primtive types in Python (int) fail in TorchScript
+   * - `6.3.2. Subscriptions <https://docs.python.org/3/reference/expressions.html#subscriptions>`_
+     - Supported
+     -
+   * - `6.3.3. Slicings <https://docs.python.org/3/reference/expressions.html#slicings>`_
+     - Partially Supported
+     - Tuple slicing with stride doesn't work
+   * - `6.3.4. Calls <https://docs.python.org/3/reference/expressions.html#calls>`_
+     - Partially Supported
+     - args unpack / kwargs unpack doesn't work
+   * - `6.4. Await expression <https://docs.python.org/3/reference/expressions.html#await-expression>`_
+     - Not Supported
+     -
+   * - `6.5. The power operator <https://docs.python.org/3/reference/expressions.html#the-power-operator>`_
+     - Supported
+     -
+   * - `6.6. Unary arithmetic and bitwise operations <https://docs.python.org/3/reference/expressions.html#unary-arithmetic-and-bitwise-operations>`_
+     - Partially Supported
+     - negate implemented only for tensors
+   * - `6.7. Binary arithmetic operations <https://docs.python.org/3/reference/expressions.html#binary-arithmetic-operations>`_
+     - Partially Supported
+     - See delimiters
+   * - `6.8. Shifting operations <https://docs.python.org/3/reference/expressions.html#shifting-operations>`_
+     - Not Supported
+     -
+   * - `6.9. Binary bitwise operations <https://docs.python.org/3/reference/expressions.html#binary-bitwise-operations>`_
+     - Supported
+     -
+   * - `6.10. Comparisons <https://docs.python.org/3/reference/expressions.html#comparisons>`_
+     - Supported
+     -
+   * - `6.10.1. Value comparisons <https://docs.python.org/3/reference/expressions.html#value-comparisons>`_
+     - Partially Supported
+     - Maybe None == None should throw an error since the ref says not to do it, list/dict equality
```

Comment:
I would remove: `Maybe None == None should throw an error since the ref says not to do it`. What do you mean by list/dict equality?